client/src/components/Notifications/NotificationDropDown.js

```diff
@@ -258,9 +314,20 @@ export const NotificationDropDown = ({
   notifications,
   organisationId,
 }) => {
-  const { markNotificationsAsRead } = useContext(WebSocketContext);
-  const dispatch = useDispatch();
+  const {
+    markNotificationsAsRead,
+    clearNotification: clearRemoteNotification,
+  } = useContext(WebSocketContext);
   const { t } = useTranslation();
+  const dispatch = useDispatch();
+
+  const clearNotification = (notificationId) => {
+    dispatch({
+      type: "LOCAL_NOTIFICATIONS_MARK_AS_CLEARED",
```

Comment:
Please import this from the wsActions

Reply:
I noticed that other dispatches don't import and directly use the string (such as `updateReadAt`), so I'd though to keep them consistent. Would you also like me to change the other ones, or just `clearNotification`?

Reply:
No I mean just import this string `"LOCAL_NOTIFICATIONS_MARK_AS_CLEARED" `as `LOCAL_NOTIFICATIONS_MARK_AS_CLEARED` since it is exported from wsActions, so if the string changed, we don't have to change it anywhere. but that's a very minor thing, you can ignore for now.